src/pages/ReimbursementAccount/BusinessInfo/BusinessInfo.js

```diff
@@ -0,0 +1,152 @@
+import IncorporationStateBusiness from './substeps/IncorporationStateBusiness';
+import ConfirmationBusiness from './substeps/ConfirmationBusiness';
+import getSubstepValues from '../utils/getSubstepValues';
+import reimbursementAccountDraftPropTypes from '../ReimbursementAccountDraftPropTypes';
+import * as ReimbursementAccountProps from '../reimbursementAccountPropTypes';
+import getInitialSubstepForBusinessInfo from '../utils/getInitialSubstepForBusinessInfo';
+
+const propTypes = {
+    /** Reimbursement account from ONYX */
+    reimbursementAccount: reimbursementAccountPropTypes,
+
+    /** The draft values of the bank account being setup */
+    reimbursementAccountDraft: reimbursementAccountDraftPropTypes,
+
+    /* The workspace policyID */
+    policyID: PropTypes.string,
+};
+
+const defaultProps = {
+    reimbursementAccount: ReimbursementAccountProps.reimbursementAccountDefaultProps,
+    reimbursementAccountDraft: {},
+    policyID: '',
+};
+
+const STEPS_HEADER_HEIGHT = 40;
+// TODO Will most likely come from different place
+const STEP_NAMES = ['1', '2', '3', '4', '5'];
+
+const bodyContent = [
+    NameBusiness,
+    TaxIdBusiness,
+    WebsiteBusiness,
+    PhoneNumberBusiness,
+    AddressBusiness,
+    TypeBusiness,
+    IncorporationDateBusiness,
+    IncorporationStateBusiness,
+    ConfirmationBusiness,
+];
+
+const businessInfoStepKeys = CONST.BANK_ACCOUNT.BUSINESS_INFO_STEP.INPUT_KEY;
+
+function BusinessInfo({reimbursementAccount, reimbursementAccountDraft, policyID}) {
+    const {translate} = useLocalize();
+
+    /**
+     * @param {Array} fieldNames
+     *
+     * @returns {*}
+     */
+    const getBankAccountFields = useCallback(
+        (fieldNames) => ({
+            ..._.pick(lodashGet(reimbursementAccount, 'achData'), ...fieldNames),
+            ..._.pick(reimbursementAccountDraft, ...fieldNames),
+        }),
+        [reimbursementAccount, reimbursementAccountDraft],
+    );
+
+    const submit = useCallback(() => {
+        const values = getSubstepValues(businessInfoStepKeys, reimbursementAccountDraft, reimbursementAccount);
+
+        const payload = {
+            bankAccountID: getDefaultStateForField({reimbursementAccount, fieldName: 'bankAccountID', defaultValue: 0}),
+            ...values,
+            ...getBankAccountFields(['routingNumber', 'accountNumber', 'bankName', 'plaidAccountID', 'plaidAccessToken', 'isSavings']),
+            companyTaxID: values.companyTaxID.replace(CONST.REGEX.NON_NUMERIC, ''),
+            companyPhone: parsePhoneNumber(values.companyPhone, {regionCode: CONST.COUNTRY.US}).number.significant,
+        };
+
+        BankAccounts.updateCompanyInformationForBankAccount(payload, policyID);
+    }, [getBankAccountFields, reimbursementAccount, reimbursementAccountDraft, policyID]);
+
+    const startFrom = useMemo(() => getInitialSubstepForBusinessInfo(lodashGet(reimbursementAccount, ['achData'], {})), [reimbursementAccount]);
```

Comment:
When getting inital substep you should check also reimbursementAccountDraft. This way if user is going through the flow for the first time, stops halfway and then goes back to it he will continue where he left off based on draft values rather than the first substep

Reply:
right, good catch 👍🏻